persistence/src/commonMain/db_user/com/wire/kalium/persistence/Conversations.sq

```diff
@@ -15,6 +15,7 @@ CREATE TABLE Conversation (
     muted_time INTEGER DEFAULT 0 NOT NULL,
     last_modified_date TEXT NOT NULL,
     last_notified_message_date TEXT,
+    last_read_date TEXT,
```

Comment:
Not for this PR but:
I think we should consider storing `last_read_date` (and all other timestamps which are always in the past) as an integer.

Reply:
will add a TODO